docs/UserGuide.md

```diff
@@ -289,6 +289,30 @@ Expected Output:
 
 ![Adding Appointment](img/output/adding_appointment.png)
 
+## Motivation Feature:
```

Comment:
Please format your userguide properly. The lines 292 to 294 are supposed to be for the appointment header. 

Refer to the format of the other portion and do accordingly

Reply:
You would have to update the help command in the constant too and the code to reflect motivation